h/static/styles/partials/_base.scss

```diff
@@ -39,6 +39,9 @@ $title-font-size: 19px;
 // Font sizes for specific categories of control
 $input-font-size: 15px;
 
+// Font size for search results "# Matching Annotations" & sidebar subtitles
+$search-results-subtitle-font-size: 15px;
```

Comment:
The other fonts defined in this file have more generic names than this because they were intended to form part of a design system in which there are only a small number of distinct font sizes ("normal", "small", "input" and "title") and colors.

@dawariley I don't need an answer on this right away, but do you have an idea of whether this font is just for this specific part of the UI or whether it will more generally used elsewhere as some kind of "heading level X" font or something like that?